### ISSUE TITLE
Lisp makes you say thhoe instead of shoe.

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#30208]
(WEB REPORT BY: eneocho REMOTE: 206.221.180.138:7777)
# Revision
0d84d10bf7318d606f78e328d8ffa6a693457846

# Description
title
# Steps to Reproduce
make a sperm cell get to the egg
# What you Expected
shoe
# What Actually Happened
somehow lisping makes you say thhoe.

**Comment by west3436** [comment]
Where is the bug here?

**Comment by boy2mantwicethefam** [comment]
feature

**Comment by Eneocho** [comment]
> 
> 
> Where is the bug here?

I've yet to hear a single lisped person say thhoe instead of shoe.

**Comment by ghost** [comment]
scunthorpe problem strikes again

**Comment by Eneocho** [comment]
> 
> 
> scunthorpe problem strikes again

That's a clbuttic.

**Comment by xeroxemnas** [comment]
we need an AUTISM tag just for moments like these.

**Comment by Eneocho** [comment]
> 
> 
> we need an AUTISM tag just for moments like these.

That's what the muh immulsions and consistency issue tags are for.

**Comment by DeityLink** [comment]
no, this is clearly different. I still have no idea what this issue is about.

**Comment by Eneocho** [comment]
> 
> 
> no, this is clearly different. I still have no idea what this issue is about.

Regex causing weird lisps that don't happen in real life and feel weird as fuck.
